stats/Export.md

```diff
@@ -28,3 +27,14 @@ parameter from the `View` and the aggregated data associated with those `columns
 * `end_time`: a timestamp, describing the end time of the current stats snapshot.
 
 The library SHOULD provide a means of retrieving the ViewData for any registered view in the system.
+
+### Aggregation to Metrics
+
+| Aggregation  | Measure Type | Metric Type  | Value Type   |
+|--------------|--------------|--------------|--------------|
+| Count        | Any          | CUMULATIVE   | INT64        |
```

Comment:
I think "Int64 or Double" would be clearer than "Any".

Reply:
Done.